Replaced old 404 image with blank frank picture and text

Angel-Armendariz opened the pull request:
Basically, I did the thingy with the thongy and used the blank Frank picture that Mike gave me with a 404 error message above it to fix the issue. It works on both mobile and desktop after doing some magic wizard stuff. I think it works alright. I apologize to Mike in advance because there is no question mark above Frank's head like in the original photo but I think he will find it in his heart to forgive me.

Angel-Armendariz committed:
Fixed Issue #417

So I used the blank lost frank image and added a 404 error message above it and made it work in mobile and desktop settings. I did not add the question mark above Frank's head like the original but it is functional.

Angel-Armendariz committed:
Merge branch 'main' of https://github.com/EthanThatOneKid/acmcsuf.com

Angel-Armendariz committed:
Update __error.svelte

@jaasonw  made the gap smaller due to suggestion

Angel-Armendariz committed:
Made the space smaller, aligned text, used default font

I changed the space to 0.5em, aligned the text center to fix the problem, and left the text as default font so it would go to Poppins by itself since I think that's our default

Angel-Armendariz committed:
Delete launch.json

accidentally included this

Angel-Armendariz committed:
Merge branch 'main' into Fix/Issue417

Angel-Armendariz committed:
404 not visible in minimized view

Added margin top in order to fix issue where text from 404 was not showing up in minimized view

Angel-Armendariz committed:
Merge branch 'Fix/Issue417' of https://github.com/EthanThatOneKid/acmcsuf.com into Fix/Issue417

jaasonw commented on src/routes/__error.svelte:
```diff
@@ -8,21 +8,33 @@
+    text-align: center;
+    font-size: 50px;
+    margin-bottom: 0.5em;
+    margin-top: 2em;
```
maybe add more to the bottom too and i think we're gucci

Angel-Armendariz replied:
> add more to*

So you want me to increase the margin-bottom? Like 1 or 2em?

jaasonw replied:
ye the picture lookin kinda cramped on the bottom too if u make the window small enough

jaasonw replied:
<img width="1174" alt="image" src="https://user-images.githubusercontent.com/8981287/168750823-ae79b211-1693-4dac-994d-21d5a58c3d6e.png">

Angel-Armendariz replied:
2em too strong? Or 1 em?
![image](https://user-images.githubusercontent.com/95111582/168757039-8f36eebc-b843-4b64-8d0c-d369c0b48a8e.png)

Angel-Armendariz replied:
Or do you prefer 3em my boy
![image](https://user-images.githubusercontent.com/95111582/168757566-a565e8d3-6e2b-4102-9520-bd23ec9fdf54.png)
, this is 3em

jaasonw replied:
3's probably fine

Angel-Armendariz replied:
Pushing now, I await your new bug that you will find soon 
![image](https://user-images.githubusercontent.com/95111582/168758903-8d74f435-3bd8-48fa-bdd3-00db7095b90f.png)

Angel-Armendariz committed:
Adding more margin to the bottom of image in minimized

Minimized view looked squished so it needed more margin on the bottom so we added 3em margin bottom to give it some space

Angel-Armendariz committed:
Merge branch 'main' into Fix/Issue417

Angel-Armendariz committed:
Merge branch 'main' into Fix/Issue417

EthanThatOneKid commented on src/routes/__error.svelte:
```diff
@@ -8,34 +8,52 @@
+    font-size: 50px;
+    margin-bottom: 0.5em;
+    margin-top: 2em;
   }
```
To make the `<em>` stand out like it does in the original 404 graphic, we could add a [text-shadow property](https://developer.mozilla.org/en-US/docs/Web/CSS/text-shadow) to this CSS block: 

Mode | Dark | Light
---|---|---
Before text-shadow | ![image](https://user-images.githubusercontent.com/31261035/169717052-76d27ac9-479a-4059-a10d-5e8e260d3807.png) | ![image](https://user-images.githubusercontent.com/31261035/169717063-49512868-a0e7-4897-9cff-ec8d2b43d2ab.png)
After text-shadow | ![image](https://user-images.githubusercontent.com/31261035/169716995-5d60ce45-d94b-4e4f-b7de-3c90720fb6cb.png) | ![image](https://user-images.githubusercontent.com/31261035/169716969-a33cc18a-abcd-4253-9bf2-64fbd92ea027.png)

```suggestion
    text-shadow: 2px 2px var(--acm-blue);
  }
```

EthanThatOneKid replied:
I used `--acm-blue` for this suggestion, but the original 404 graphic used `#92c4df`.

![image](https://user-images.githubusercontent.com/31261035/169718924-13188ca7-6849-4636-9e39-d3835c278afa.png)

EthanThatOneKid replied:
> I used `--acm-blue` for this suggestion, but the original 404 graphic used `#92c4df`.
> 
> ![image](https://user-images.githubusercontent.com/31261035/169718924-13188ca7-6849-4636-9e39-d3835c278afa.png)

To better match the original 404 graphic, you are able to add some more weight, lighten the color, and increase the font size as needed (while keeping in mind how the element responds on mobile and wide screens).
```suggestion
    /* #92c4df is slightly lighter than var(--acm-blue) */
    /* see <https://etok.codes/acmcsuf.com/pull/458/files#r878929505> */
    text-shadow: 2px 2px #92c4df;
    font-weight: 600;
  }
```

Angel-Armendariz replied:
Done

EthanThatOneKid commented on src/routes/__error.svelte:
```diff
@@ -8,34 +8,52 @@
+    align-items: center;
+    text-align: center;
+  }
+  section em {
```
To make the CSS easier to read, we add a blank line separating each CSS definition.
```suggestion
  }
  
  section em {
```

Angel-Armendariz replied:
done

EthanThatOneKid commented on src/routes/__error.svelte:
```diff
@@ -8,34 +8,52 @@
+  <em>404</em>
+  <h1>Frank can't find where you're going!</h1>
+  <img src="/assets/png/lost-frank.png" class="wide" alt="404 - Page Not Found" />
+  <img src="/assets/png/lost-frank.png" class="mobile" alt="404 - Page Not Found" />
```
I noticed that the `wide` and `mobile` class names are not in use, so you can go ahead and remove one of the images.
```suggestion
  <img src="/assets/png/lost-frank.png" alt="404 - Page Not Found" />
```

Angel-Armendariz replied:
done

EthanThatOneKid commented on src/routes/__error.svelte:
```diff
@@ -8,34 +8,52 @@
+    section .mobile {
+      display: none;
+    }
   }
```
Unless I misunderstand, the `wide` and `mobile` class names seem to not affect anything. If so, you can go ahead and remove these lines of CSS.
```suggestion
```

Angel-Armendariz replied:
done

EthanThatOneKid reviewed (dismissed):
Congrats on finishing up with finals; _phew_! Managed to take a look at your PR again and added some minor suggestions. Progress looks great so far! Thanks for contributing :)

Angel-Armendariz committed:
Update src/routes/__error.svelte

Co-authored-by: Ethan Davidson <31261035+EthanThatOneKid@users.noreply.github.com>

Angel-Armendariz committed:
Update src/routes/__error.svelte

Co-authored-by: Ethan Davidson <31261035+EthanThatOneKid@users.noreply.github.com>

Angel-Armendariz committed:
Update src/routes/__error.svelte

Co-authored-by: Ethan Davidson <31261035+EthanThatOneKid@users.noreply.github.com>

Angel-Armendariz committed:
Update src/routes/__error.svelte

Co-authored-by: Ethan Davidson <31261035+EthanThatOneKid@users.noreply.github.com>

Angel-Armendariz committed:
Changes have been made that were requested

Changes look good and the fix looks like it works in desktop, mobile, and minimized view.

Angel-Armendariz committed:
Added suggested changes

text shadow was made lighter as suggested and font-weight was added, suggested by @EthanThatOneKid

Angel-Armendariz committed:
Merge branch 'main' into Fix/Issue417
